### ISSUE TITLE
Wayland Support

### ISSUE THREAD

**Issue by cd-CreepArghhh** [f3d-app/f3d#1375]
TL;DR runs on Xwayland, is it possible to get it working on Wayland?

### Is your feature request related to a problem? Please describe.
Currently, F3D runs under Xwayland. This means it can't fractionally scale. Note the text:
![Screenshot of title in F3D](https://github.com/f3d-app/f3d/assets/65649991/c9621d35-818a-47b3-a2e4-2fedb83467b4)
![Screenshot of the axis in F3D](https://github.com/f3d-app/f3d/assets/65649991/9f3f3b80-a38d-45b4-8e26-0bb43e3d23b5)
Also, since it can't fractionally scale, my cursor appears to be 1.5x larger than it should.
In F3D:
![image](https://github.com/f3d-app/f3d/assets/65649991/7e9a4dd5-63cc-4b02-89cf-94244b7d32ba)
Everywhere else:
![image](https://github.com/f3d-app/f3d/assets/65649991/6f5d3092-4a3f-4f75-95f5-ff2769a70b73)
This is simply an X11 problem.

### Describe the solution you'd like
Figure out how to use Wayland with whatever graphics library is being used (not sure if this is a Vtk problem).

### Describe alternatives you've considered
  - Not do anything.
  - Lobby for Xwayland or X11 or whatever to support fractional scaling (<-- not happening).
  - If this is an issue with Vtk (which I assume does the rendering), create a feature request upstream, then wait.

### Additional information
There might be some cute little command line flags or environment flags that the GUI library supports to enable Wayland. Not sure.

### Contribution
**Please tell us if you are able to contribute (Can you develop this new feature?)**
Nope, I have no idea on anything graphics library related, except that there's a couple flags you can set for Qt, GTK and Electron apps.

**Comment by mwestphal** [comment]
HI @cd-CreepArghhh 

> not sure if this is a Vtk problem

It is a VTK issue: https://gitlab.kitware.com/vtk/vtk/-/issues/17982

**Comment by cd-CreepArghhh** [comment]
Oh right, should I close this then?

**Comment by mwestphal** [comment]
No, this issue is still valid, it just that the "fix" (its an huge feature actually, not a simple fix) must happen in VTK at some point.

**Comment by mwestphal** [comment]
Wayland rendering is currently being added to VTK! 
https://gitlab.kitware.com/vtk/vtk/-/merge_requests/12107

**Comment by Otzie2023** [comment]
@mwestphal The problem still exists; it was tested in the latest nightly build (F3D-3.5.0-255-g1c2f71a7-Linux-x86_64.deb) on Ubuntu26.  In F3D, text and other graphic elements are therefore displayed very blurrily.

**Comment by mwestphal** [comment]
> [@mwestphal](https://github.com/mwestphal) The problem still exists; it was tested in the latest nightly build (F3D-3.5.0-255-g1c2f71a7-Linux-x86_64.deb) on Ubuntu26. In F3D, text and other graphic elements are therefore displayed very blurrily.

Well, the issue is not closed is it ? :)

While wayland support is now in VTK, nothing was integrated in F3D yet.
